task/gogit/0.1/gogit.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: gogit
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/categories: CLI
+    tekton.dev/tags: search
+    tekton.dev/platforms: "linux/amd64"
+spec:
+  description: >-
+    Send a build status to Gitlab/GitHub.
+  params:
+  - name: provider
+    description: provider name of git
+    type: string
+    default: github
+  - name: repo
+    description: git repository name
+    type: string
+  - name: username
+    description: username of git provider
+    type: string
+  - name: token
+    description: personal access token of git provider
+    type: string
+  - name: pr
```

Comment:
can you please rename this parameter name from `pr` to something else like `pr_number` or `pull_request_number` as `pr` seems to be very vague